### PR TITLE
ZF3 compatibility

### PR DESCRIPTION
**Pull request by webimpress**
In difference to #564 this PR add ZF3 compatibility but also __keep ZF2 compatibility__.

__There is no any BC breaks ! ! !__ 😄

Other changes introduced in #564 could be done later.

The only problem was with `zend-mvc-console` (over ZF3) so please see suggest section in `composer.json` and travis configuration.

Thoughts?

### CODE REVIEW

**Comment by shandyDev** [comment]
I suggest add tag (release) to current master, 1.2.0 (support both ZF v2 and v3). And for BC break in https://github.com/doctrine/DoctrineModule/pull/564 and future drop support ZFv2 in 2.x releases.

**Comment by Slamdunk** [comment]
@gianarb may you tag the current master, perhaps with a non-stable one ([alpha, beta or RC](https://getcomposer.org/doc/04-schema.md#minimum-stability))?

**Comment by gianarb** [comment]
I am in holiday :) Give my 10 days and I come back

**Comment by Slamdunk** [comment]
@gianarb was your holiday good? :stuck_out_tongue:

**Comment by gianarb** [comment]
👍  too much!

File: src/DoctrineModule/Stdlib/Hydrator/DoctrineObject.php [hunk 1]
```diff
@@ -27,8 +27,8 @@
 use RuntimeException;
 use Traversable;
 use Zend\Stdlib\ArrayUtils;
-use Zend\Stdlib\Hydrator\AbstractHydrator;
```
Comment (svycka) [inline src/DoctrineModule/Stdlib/Hydrator/DoctrineObject.php]: how this is not BC break?

**Pull request merged**